frontend/src/tests/components/Sections/SectionsTable.test.js

```diff
@@ -88,7 +88,7 @@ describe("UserTable tests", () => {
         });
 
         await waitFor( ()=> expect(screen.getByTestId(`${testId}-cell-row-0-col-courseInfo.courseId`)).toHaveTextContent("CMPSC 8"));
-        expect(screen.getByTestId(`${testId}-cell-row-1-col-courseInfo.courseId`)).toHaveTextContent("CMPSC 9");
+        expect(screen.getByTestId(`${testId}-cell-row-1-col-quarter`)).toHaveTextContent("F20");
```

Comment:
Instead of editing this line to check for quarter instead of courseId, can we keep both lines? It would be helpful to test for both of these conditions.